MODULE.bazel

```diff
@@ -0,0 +1,63 @@
+"""Module Declaration for rules_ros2."""
+
+module(
+    name = "rules_ros2",
```

Comment:
Anyone can make a workspace/module rules_ros2. I gave kind-of unique name com_github_mvukov_rules_ros2 to distinguish this work from possible other attempts to hook ROS 2 to Bazel. Thus, please stick with com_github_mvukov_rules_ros2. If it turns out rules_ros2 would be sufficient/desired, we can always have a follow-up PR just to rename stuff. Let's have this PR as simple and minimal as possible.

Reply:
Yeah I understand that, but you can alias your `MODULE.bazel` to this, 

```bazel
bazel_dep ( name = "rules_ros2" , version = "0.1.0" , repo_name = "com_github_mvukov_rules_ros2 ") 
```

it would be convenient for a user to call in `BUILD.bazel` files is what I thought but sure.

Reply:
Since we're going to continue to use the workspace-based workflow for a while, I'd prefer the name to be the same; hence: com_github_mvukov_rules_ros2.

Reply:
sure will keep the module name the same